src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -61,9 +61,41 @@ trait MarathonConf extends ScallopConf with ZookeeperConf with IterativeOfferMat
   def executor: Executor = Executor.dispatch(defaultExecutor())
 
   lazy val mesosRole = opt[String]("mesos_role",
-    descr = "Mesos role for this framework",
+    descr = "Mesos role for this framework. " +
+      "If set, Marathon receives resource offers for the specified role in addition to " +
+      "resources with the role designation '*'.",
     default = None)
 
+  def expectedResourceRoles: Set[String] = mesosRole.get match {
+    case Some(role) => Set(role, "*")
+    case None       => Set("*")
+  }
+
+  lazy val defaultAcceptedResourceRolesSet = defaultAcceptedResourceRoles.get.get
+
+  lazy val defaultAcceptedResourceRoles = opt[String]("default_accepted_resource_roles",
+    descr =
+      "Default for the defaultAcceptedResourceRoles attribute of all app definitions" +
+        " as a comma-separated list of strings." +
+        "This defaults to all roles for which this Marathon instance is configured to receive offers.",
+    default = Some(expectedResourceRoles.mkString(",")),
```

Comment:
Good that you noticed that.

Fixed now. I also added an integration test for that.